src/script/team/TeamRepository.ts

```diff
@@ -236,6 +236,21 @@ export class TeamRepository extends TypedEventEmitter<Events> {
     await this.updateTeamMembersByIds(selfTeamId, newTeamMemberIds, true);
   };
 
+  filterRemoteDomainUsers(users: User[]): User[] {
+    const isMLS = this.getTeamSupportedProtocols().includes(ConversationProtocol.MLS);
```

Comment:
still feels like we should find a way to get the default protocol instead of is MLS is supported.
The tickets read like we should filter fed users if Proteus is the default protocol even if MLS is supported

Reply:
for reference, that's the check in the group creation modal
```
  //if feature flag is set to false or mls is disabled for current team use proteus as default
  const defaultProtocol =
    isMLSFeatureEnabled && isMLSEnabledForTeam
      ? teamState.teamFeatures()?.mls?.config.defaultProtocol
      : ConversationProtocol.PROTEUS;